service/cziti/sdk.go

```diff
@@ -656,68 +706,25 @@ func eventCB(ztx C.ziti_context, event *C.ziti_event_t) {
 				servicesToAdd = append(servicesToAdd, svcToAdd)
 			}
 		}
-		var minimumTimeout int32 = -1
-		var maximumTimeout int32 = -1
-		var minimumTimeoutRem int32 = -1
-		var maximumTimeoutRem int32 = -1
-		noTimeoutSvc := false
-		noTimeoutRemSvc := false
-		if len(servicesToAdd) > 0 {
-			zid.Services.Range(func(key interface{}, value interface{}) bool {
-				//string, ZService
-				val := value.(*ZService)
-
-				if val.Service.Timeout >= 0 {
-					if minimumTimeout == -1 || minimumTimeout > val.Service.Timeout {
-						minimumTimeout = val.Service.Timeout
-					}
-					if maximumTimeout == -1 || maximumTimeout < val.Service.Timeout {
-						maximumTimeout = val.Service.Timeout
-					}
-				} else {
-					noTimeoutSvc = true
-				}
-				if val.Service.TimeoutRemaining >= 0 {
-					if minimumTimeoutRem == -1 || minimumTimeoutRem > val.Service.TimeoutRemaining {
-						minimumTimeoutRem = val.Service.TimeoutRemaining
-					}
-					if maximumTimeoutRem == -1 || maximumTimeoutRem < val.Service.TimeoutRemaining {
-						maximumTimeoutRem = val.Service.TimeoutRemaining
-					}
-				} else {
-					noTimeoutRemSvc = true
-				}
-
-				return true
 
-			})
-			if noTimeoutSvc {
-				maximumTimeout = -1
-			}
-			if noTimeoutRemSvc {
-				maximumTimeoutRem = -1
-			}
+		if len(servicesToAdd) > 0 || len(servicesToRemove) > 0 {
```

Comment:
why not do this for changed services too? shouldn't this be for 'all service changes'?

Reply:
changed services are already copied to the servicesToRemove and servicesToAdd lists